api/relations/v0/relation_tuples.proto

```diff
@@ -0,0 +1,76 @@
+syntax = "proto3";
+
+package kessel.relations.v0;
+
+import "google/api/annotations.proto";
+import "relations/v0/common.proto";
+
+option go_package = "ciam-rebac/api/relations/v0;v0";
+option java_multiple_files = true;
+option java_package = "org.projectkessel.api.relations.v0";
+
+// KesselTupleServices manages the persisted _Tuples_ stored in the system..
+// 
+// A Tuple is an explicitly stated, persistent relation 
+// between a Resource and a Subject or Subject Set. 
+// It has the same _shape_ as a Relationship but is not the same thing as a Relationship.
+// 
+// A single Tuple may result in zero-to-many Relationships.
+service KesselTupleService {
```

Comment:
Why not RelationshipTupleService or even just TupleService? The whole API is already scoped under Kessel I would have thought.